pkgs/applications/science/machine-learning/torch/torch-distro.nix

```diff
@@ -0,0 +1,358 @@
+{ pkgs ? import <nixpkgs> {}
```

Comment:
I am trying to keep the ability of using torch-distro.nix as a standalone build script. I think, pkgs could be easily replaced with explicit deps, but `src` argument is still important for me.